rust_src/remacs-sys/lib.rs

```diff
@@ -23,6 +23,10 @@ pub mod libm;
 use libc::{c_char, c_double, c_float, c_int, c_short, c_uchar, c_void, intmax_t, off_t, ptrdiff_t,
            size_t, time_t, timespec};
 
+// libc refuses to merge pid_t as an alias for c_int in Windows, so we will not use libc::pid_t
```

Comment:
Perhaps 'prefers not to merge'? 'Refuses' sounds like we think they're being unreasonable. It's a shame, but I think their position is defensible, and being nice makes it easier to file bug reports in future :)

Reply:
Good suggestion, will update.